index.html

```diff
@@ -4017,26 +4017,11 @@
 
 		localsettings.opmode = temp_scenario.opmode;
 		if (temp_scenario.opmode == 3) {
-			if (temp_scenario.enhanced_chat_ui===true) {
-				localsettings.enhanced_chat_ui = true;
-			}
-			else if(temp_scenario.enhanced_chat_ui===false)
-			{
-				localsettings.enhanced_chat_ui = false;
-			}
-			if (temp_scenario.multiline_replies===true) {
-				localsettings.multiline_replies = true;
-			}
-			else if(temp_scenario.multiline_replies===false)
-			{
-				localsettings.multiline_replies = false;
-			}
-			if (temp_scenario.chatopponent) {
-				localsettings.chatopponent = temp_scenario.chatopponent;
-			}
-			if (temp_scenario.chatname) {
-				localsettings.chatname = temp_scenario.chatname;
-			}
+			localsettings.enhanced_chat_ui = temp_scenario.enhanced_chat_ui;
+			localsettings.multiline_replies = temp_scenario.multiline_replies;
+			
+			if (temp_scenario.chatopponent) { localsettings.chatopponent = temp_scenario.chatopponent; }
+			if (temp_scenario.chatname) { localsettings.chatname = temp_scenario.chatname; }
```

Comment:
@Lyrcaxis 
I understand you are trying to simplify the code, but this is incorrect. The reason it needs to be handled like that is because there is a third state - non-existent value, which can happen for a variety of fields when loading for old format files. Assigning `null` to `localsettings.enhanced_chat_ui` would be a very bad idea, hence the **Strict Equality Check** `===` to eliminate Truthy and Falsy values.